generator/src/test/java/com/scottlogic/deg/generator/restrictions/DateTimeRestrictionsTests.java

```diff
@@ -52,39 +55,6 @@ public void shouldBeEqualIfMinAndMaxMatchOther(){
         Assert.assertThat(a.hashCode(), equalTo(b.hashCode()));
     }
 
-    @Test
```

Comment:
Have equivalent tests like this been created for the new implementations/behaviour?